nxc/helpers/bloodhound.py

```diff
@@ -43,13 +43,21 @@ def add_user_bh(user, domain, logger, config):
         try:
             with driver.session() as session, session.begin_transaction() as tx:
                 for info in users_owned:
+                    distinguished_name = "".join(["DC=" + dc + "," for dc in info["domain"].split(".")]).rstrip(",")
+                    domain_query = tx.run(f'MATCH (d:Domain) WHERE d.distinguishedname STARTS WITH "{distinguished_name}" RETURN d').data()
```

Comment:
Does the inner quotes have to be double? I know in some cases that's a necessity due to the query language (or like with Windows sometimes).

Reply:
Nope, that was just Code with the old style which i copied lol. Will change that